Makefile

```diff
@@ -56,7 +57,7 @@ test:
 test-all:
 	tox
 
-assets: staticdeps
+assets:
```

Comment:
I removed `staticdeps` from `make assets` because I wanted to be able to quickly build just the "assets" as in JS, CSS etc without getting `kolibri/dist` populated... it's confusing once it's populated because when installing other versions with `pip install` or switching branches, the contents of this folder remain unchanged and override whatever is installed in the `virtualenv`.